### ISSUE TITLE
Component does not render anything

### ISSUE THREAD

**Issue by AleVul** [unicef/react-org-chart#17]
Managed to get to compile after some rather weird bugs such as `loadImage` prop being required (it crashes if not set) in situation when it should, like setting image src in the tree objects.

But no mather what i do, this renders 2 divs that are hidden and an svg that doent output anything.

Also this seems to get a handle on a element with id `root`. Nowhere this is specified, thus crashes when the element was not found.

Any suggestion what i can do to get this render properly?

P.S.: is this repository abandoned?

**Comment by Boris-Scharinger** [comment]
Indeed, there are several forks from this repository respectively from the coredev version underneath that are 'alive', this isn't.

**Comment by itsnotme01** [comment]
Hello, 
I finally manage to display the tree by editing the svg width and height in my web browser.
I now need to edit it in the code to perenise the fix.

Before edition:
![OrgChartSvg](https://user-images.githubusercontent.com/23737696/90534047-8f518c00-e179-11ea-8a97-a4e3aa6c6fa6.png)

After:
![OrgChartSvgEdited](https://user-images.githubusercontent.com/23737696/90534048-8fea2280-e179-11ea-8da6-9c6a21afad63.png)

**Comment by Yalian** [comment]
@itsnotme01  can you solve it?

**Comment by tusharverma200** [comment]
Hi @Yalian, I can solve this issue. Assign this to me.
